lightning_pose/utils/scripts.py

```diff
@@ -756,4 +756,5 @@ def export_predictions_and_labeled_video(
             ys_arr=ys_arr,
             mask_array=mask_array,
             filename=labeled_mp4_file,
+            colormap=cfg.eval.colormap
```

Comment:
update to `colormap=cfg.eval.get("colormap", "cool")` in case someone runs this with an older config that doesn't have the `colormap` option

Reply:
Good point. I was wondering how one resolves this issue. Will upodate it :)